docs/readthedocs/about.md

```diff
@@ -1,10 +1,32 @@
 (about)=
 # About SIMONA
 
-## Overview
+SIMONA is a simulation tool for generating time series in the context of distribution grids to be used in grid planning, 
+operation and analysis. The simulation is executed using an agent-based, discrete-event model. SIMONA is being developed
+at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at
+[TU Dortmund University](https://www.tu-dortmund.de), Germany.
 
-SIMONA is an agent-based discrete-event power system simulation model developed at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at [TU Dortmund University](https://www.tu-dortmund.de), Germany.
+## Power Grid Model
+With SIMONA, you can model your power grid using a broad variety of different available asset types. The input data for
+the grid must be provided in the PowerSystemDataModel format. (For more information on this data model, please visit the
+{doc}`PowerSystemDataModel <psdm:index>` documentation.) The combination of implemented
+physical participant models and your provided input data forms a power grid that can be further investigated and 
+analysed using ancillary tools.
 
-## Further information and Contact Details
+![Basic Simona](images/usersguide/basic_simona_environment.png "SIMONA System Overview")
```

Comment:
I think the alt text is the one in the beginning. So I would place the `SIMONA System Overview` text where currently `Basic Simona` is placed.